website/docs/installation/documentdb/kubernetes.md

```diff
@@ -0,0 +1,74 @@
+---
+sidebar_position: 3
+---
+
+# Kubernetes
+
+FerretDB uses PostgreSQL with [DocumentDB extension](https://github.com/microsoft/documentdb) as a database engine.
+You need to have a running Kubernetes cluster to set up PostgreSQL with DocumentDB extension on Kubernetes.
```

Comment:
Why do we say everything (PostgreSQL, DocumentDB, Kubernetes) twice in two sentences in two paragraphs?

Reply:
It still repeats PostgreSQL and DocumentDB. Can it be simplified?

Reply:
That's not repeated. It's simplified unless you're suggesting I remove "PostgreSQL" and just leave it as DocumentDB.

Reply:
Not a big deal if we repeat _PostgreSQL with DocumentDB extension_, I guess.

Reply:
Resolved the comments. Thanks for all the reviews 🤗